projects/packages/jetpack-mu-wpcom/src/features/wpcom-comments/wpcom-comments.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Plugin Name: Jetpack MU WPCom Comment Likes
+ * Description: Adds a "Like" action to comment rows and enqueues the necessary assets in the admin area.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+if ( ! defined( 'IS_WPCOM' ) || ! IS_WPCOM ) {
+	return;
+}
+
+/**
+ * Adds a "liked" class to comments that the current user has liked.
+ *
+ * @param array  $classes    Array of comment classes.
+ * @param string $css_class  Unused.
+ * @param int    $comment_id The comment ID.
+ * @return array Modified array of comment classes.
+ */
+function wpcom_comments_add_like_class( $classes, $css_class, $comment_id ) {
+	$blog_id = get_current_blog_id();
+	if ( Likes::comment_like_current_user_likes( $blog_id, $comment_id ) ) {
+		$classes[] = 'liked';
+	}
+	return $classes;
+}
+add_filter( 'comment_class', 'wpcom_comments_add_like_class', 10, 3 );
+
+/**
+ * Adds "Like" and "Unlike" action buttons to comment rows.
+ *
+ * @param array      $actions Array of actions for the comment.
+ * @param WP_Comment $comment The comment object.
+ * @return array Modified actions array.
+ */
+function wpcom_comments_enable_likes( $actions, $comment ) {
+	$actions['like'] = sprintf(
+		'<button class="button-link comment-like-button" data-comment-id="%d" aria-label="%s">%s</button>',
+		$comment->comment_ID,
+		esc_attr__( 'Like this comment', 'jetpack-mu-wpcom' ),
+		esc_html__( 'Like', 'jetpack-mu-wpcom' )
+	);
+
+	$actions['unlike'] = sprintf(
+		'<button class="button-link comment-unlike-button" data-comment-id="%d" aria-label="%s">%s</button>',
+		$comment->comment_ID,
+		esc_attr__( 'Unlike this comment liked by you', 'jetpack-mu-wpcom' ),
```

Comment:
This reads a little awkwardly, maybe just "Unlike this comment"? Or perhaps "Stop liking this comment".

Reply:
What do you think of "Remove your like from this comment"? I'm leaning toward either "Unlike this comment" or "Remove your like from this comment," but since I don't use a screen reader, I'm not sure which one is better for accessibility.

Reply:
I'm not an expert, but I'd go for the briefest thing that still makes sense in context.

Reply:
I'd favour "Unlike this comment". I agree that the "by you" is awkward, but I was wondering if we could subtly add a cue for users who might be a bit confused about what the button does or what the current state is. Probably overthinking. :)